### PR TITLE
Add events for TWIR 454

### PR DESCRIPTION
**Pull request by mariannegoldin**
Adding events through end of August in new virtual + city format, correcting some minor stuff from the last edition.

### CODE REVIEW

File: draft/2022-08-03-this-week-in-rust.md [hunk 1]
```diff
@@ -124,44 +114,41 @@ Rusty Events between 2022-08-03 - 2022-08-31 🦀
...
 * 2022-08-23 | Toronto, ON, CA | [Rust Toronto](https://www.meetup.com/rust-toronto/)
     * [**WebAssembly plugins in Rust**](https://www.meetup.com/rust-toronto/events/287284601/)
+* 2022-08-25 | Ciudad de México, MX | [Rust MX](https://www.meetup.com/rust-mx/)
+    * [**Concurrencia & paralelismo con Rust**](https://www.meetup.com/rust-mx/events/287561814/)
+* 2022-08-25 | Lehi, UT, US | [Utah Rust](https://www.meetup.com/utah-rust/)
+    * [**Hello World Cargo Crates Using Github Actions with jojobyte and Food!**](https://www.meetup.com/utah-rust/events/kvrxqsydclbpb/)
+
+### Europe
+
+* 202-08-30 | Utrecht, NL | [Rust Nederland](https://www.meetup.com/rust-nederland/)
```
Comment (andrewpollack) [inline draft/2022-08-03-this-week-in-rust.md:147]:
```suggestion
* 2022-08-30 | Utrecht, NL | [Rust Nederland](https://www.meetup.com/rust-nederland/)
```

**Review by andrewpollack — CHANGES_REQUESTED** [review]
Can we shift the "Europe" section to come after "Asia" and before "North America"? Done to help keep a rough alphabetical (even though "Virtual" comes first)

File: draft/2022-08-03-this-week-in-rust.md (hunk 1 above)
Comment (mariannegoldin) [reply draft/2022-08-03-this-week-in-rust.md]: Thanks for catching this! Will include this with the re-ordering change

File: draft/2022-08-03-this-week-in-rust.md [hunk 2]
```diff
@@ -86,24 +86,14 @@ Rusty Events between 2022-08-03 - 2022-08-31 🦀
...
 * 2022-08-03 | Virtual (Indianapolis, IN, US) | [Indy Rust](https://www.meetup.com/indyrs/)
     * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/qwtdjsydclbfb/)
 * 2022-08-03 | Virtual (Stuttgart, DE) | [Rust Community Stuttgart](https://www.meetup.com/Rust-Community-Stuttgart/)
     * [**Rust-Meetup**](https://www.meetup.com/rust-community-stuttgart/events/dvvtvsydclbfb/)
 * 2022-08-05 | Virtual + Portland, OR, US | [RustConf](https://rustconf.com/)
     * [**RustConf 2022**](https://rustconf.com/)
+* 2022-08-09 | Virtual (Myrtle Point, OR, US) | [#EveryoneCanContribute Cafe](https://www.meetup.com/everyonecancontribute-cafe/)
+    * [**Summer Chill & Learn: including OpenTelemetry getting started with Rust**](https://www.meetup.com/everyonecancontribute-cafe/events/286609523/)
 * 2022-08-09 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/Dallas-Rust/)
     * [**Second Tuesday**](https://www.meetup.com/dallas-rust/events/vndgwsydclbmb/)
```
[the remark this answers is not in the document]
Comment (mariannegoldin) [reply draft/2022-08-03-this-week-in-rust.md:98]: Going to include this in my updates of this PR, thank you so much for catching!

**Pull request merged**